CHANGELOG.md

```diff
@@ -1,8 +1,29 @@
-# dbt_youtube_analytics v0.UPDATE.UPDATE
+# dbt_youtube_analytics v0.4.0
+
+## 🚨 Breaking Changes 🚨 (within the upstream [dbt_youtube_analytics_source v0.4.0 release](https://github.com/fivetran/dbt_youtube_analytics_source/releases/tag/v0.4.0)):
+- Removed support for the `video_metadata` Cloud Function generated source table and downstream models. This also means the following variables are no longer used or necessary within the package: ([PR #12](https://github.com/fivetran/dbt_youtube_analytics_source/pull/12))
+  - `youtube__using_video_metadata`
+  - `youtube_metadata_schema`
+  - `youtube_analytics_database`
+- To be consistent with our other packages, the identifier variables have been updated. Please see the following changes to the identifier variables used in this package. ([PR #12](https://github.com/fivetran/dbt_youtube_analytics_source/pull/12))
+
+| **old identifier name** | **new identifier name** |
+| ------------------------|-------------------------|
+| `youtube__channel_basic_table` | `youtube_analytics_channel_basic_a_2_identifier` |
+| `youtube__channel_demographics_table` | `youtube_analytics_channel_demographics_a_1_identifier` |
+
+## Feature Updates:
+- Following the latest update to **all** Fivetran YouTube Analytics connectors, the `video` metadata is now synced as part of the connector by default. Therefore, the `video` source and staging models along with the following downstream models have been updated to reference the `video` source table which is synced by default: ([PR #13](https://github.com/fivetran/dbt_youtube_analytics/pull/13))
+  - `youtube__video_metadata`
```

Comment:
Yeah I felt since there were so many additional fields it would be a bit nauseating to see the full list of fields in the CHANGELOG when it wasn't really necessary.